website/src/css/_showcase-link.scss

```diff
@@ -0,0 +1,19 @@
+// Header for showcase page
+.header-showcase-link:after {
```

Comment:
Wait a sec, do we have a showcase page here, on the website?

Reply:
It’s all fun, but hold on... where can you find custom CSS here? Are you absolutely certain that this file is necessary? This page looks like a supertypical Docusaurus one... 🤔 

![image](https://github.com/user-attachments/assets/3b0e1888-4afc-4694-b081-ab32fffc93cd)